<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToeMainWindow</class>
 <widget class="QMainWindow" name="TicTacToeMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TicTacToeMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter Server Listening Port</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEditPort">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>5001</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStart">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>321</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter Remote PC Details</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEditRemoteIPAddr">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditRemotePort">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>5002</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Result | Status</string>
    </property>
    <widget class="QLabel" name="labelResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>36</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Game Not Started</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionExit"/>
    <addaction name="actionnew"/>
    <addaction name="actionFile"/>
    <addaction name="actioncopy"/>
    <addaction name="actiondocument"/>
    <addaction name="actionsave"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menudebug">
    <property name="title">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSource_Code"/>
    <addaction name="actionmore_detials"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menudebug"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnew"/>
   <addaction name="actionFile"/>
   <addaction name="actionsave"/>
   <addaction name="actioncopy"/>
   <addaction name="actiondocument"/>
  </widget>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSource_Code">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionnew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Downloads/file.png</normaloff>:/images/Downloads/file.png</iconset>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Downloads/copy.png</normaloff>:/images/Downloads/copy.png</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
  </action>
  <action name="actiondocument">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Downloads/size.png</normaloff>:/images/Downloads/size.png</iconset>
   </property>
   <property name="text">
    <string>size</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Downloads/save.png</normaloff>:/images/Downloads/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionmore_detials">
   <property name="text">
    <string>more detials</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>TicTacToeMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Game</sender>
   <signal>triggered()</signal>
   <receiver>pushButtonConnect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>826</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
